src/ProjectSystem/Impl/IO/MsBuildFileSystemWatcher.cs

```diff
@@ -215,6 +215,9 @@ public sealed partial class MsBuildFileSystemWatcher : IDisposable {
 
             relativePath = PathHelper.MakeRelative(rootDirectory, fullPath);
             try {
+                if (!fileSystem.FileExists(fullPath)) {
```

Comment:
Sure, but it is catched and method exists with false, so there can't be a popup message with exception.
I think that without debugger race happens in a different place - between this method and  `ToShortRelativePath` calls. This method returns `true`, but  `ToShortRelativePath` throws exception.